src/plugins/action_console_dummy_plugin_1/src/plugin.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "action_plugin",
+  "code": "dum1",
+  "name": "Dummy plug-in #1",
+  "file": "dummy1",
+  "function_name": "dummy1",
+  "description": "This does nothing. It's just for testing purposes.",
+  "required_parameters": ["application_id", "status"],
+  "info": {
+    "author": {
+      "name": "The mSupply Foundation",
+      "url": "http://msupply.foundation"
+    },
+    "links": [],
+    "screenshots": [],
+    "version": "1.0.0",
+    "updated": "2020-08-27"
+  }
+}
```

Comment:
Very cool! Note: if we don't have a documentation for this one yet. It will be great to add to the **public docs** reminder.

Reply:
You mean the plugin spec?

Reply:
Yep

Reply:
[Issue](#21 ) made :)